### PR TITLE
Fix color bugs

### PR DESCRIPTION
**Pull request by elenatorro**
Related Issue #13905
Project: CartoDB/product#54

TODO:
- [x] Hide panes when there is only one option (fixed or by value) to select.
- [x] Fix color when aggregation is animated (torque). There are two related issues happening now in production (#13975 and #13976) --> **We'll fix that separately.**
- [x] Hide image option when aggregation does not allow it.
- [x] Call adjustImageSize function when input color changes.

### CODE REVIEW

**Comment by elenatorro** [comment]
retest this please

File: app/assets/stylesheets/editor-3/_forms.scss [hunk 1]
```diff
@@ -8,23 +8,23 @@
...
 }
 
 .Editor-formInner.Editor-formInner--Size,
 .Editor-formInner.Editor-formInner--FillColor {
   align-items: baseline;
-}
 
-.Editor-formInner.Editor-formInner--Size.Editor-formInner--NoTabs {
-  align-items: center;
+  &.Editor-formInner--NoTabs {
```
Comment (VictorVelarde) [inline app/assets/stylesheets/editor-3/_forms.scss]: Great!

File: lib/assets/javascripts/builder/components/form-components/editors/fill/input-color/input-color-value-content-view.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ var FillConstants = require('builder/components/form-components/_constants/_fill
 var checkAndBuildOpts = require('builder/helpers/required-opts');
 var MetricsTracker = require('builder/components/metrics/metrics-tracker');
 
+var StackViewSteps = {
```
Comment (VictorVelarde) [inline lib/assets/javascripts/builder/components/form-components/editors/fill/input-color/input-color-value-content-view.js]: Much better

File: lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/fill-color.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ var StyleFormDictionaryHelpers = require('builder/editor/style/style-form/style-
 var FillConstants = require('builder/components/form-components/_constants/_fill');
 var StyleConstants = require('builder/components/form-components/_constants/_style');
 
-var NO_TABS_CLASS = 'Editor-formInner--NoTabs';
+var NO_PANES_CLASS = 'Editor-formInner--NoTabs';
```
Comment (VictorVelarde) [inline lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/fill-color.js]: It's good to see that this approach fits both parts

**Comment by elenatorro** [comment]
🚀 👏 🔥

**Pull request merged**